sovrin_common/roles.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum, unique
+
+from plenum.common.roles import Roles
+
+
+@unique
+class Roles(Enum):
+    #  These numeric constants CANNOT be changed once they have been used,
+    #  because that would break backwards compatibility with the ledger
+    TRUSTEE = Roles.TRUSTEE.value
+    TGB = "1"
+    STEWARD = Roles.STEWARD.value
+    TRUST_ANCHOR = "3"
```

Comment:
If there are any new roles added to plenum.common.roles.Roles, then the Roles class here will be broken. It may be good to update the comment here and to have values in this enum start with 5, for example--to allow future additions in plenum.